.circleci/config.yml

```diff
@@ -35,24 +35,40 @@ jobs:
   test:
     executor: pajbot
     steps:
-      - create_setup
+      - create_python_setup
       - run:
           name: Run tests
           command: ./venv/bin/pytest
   lint:
     executor: pajbot
     steps:
-      - create_setup
+      - create_python_setup
       - run:
           name: Run flake8 (lint)
           command: ./venv/bin/flake8
-  check_format:
+  check_python_format:
     executor: pajbot
     steps:
-      - create_setup
+      - create_python_setup
       - run:
           name: Run black in check mode
           command: ./venv/bin/black --check --diff pajbot alembic scripts
+  check_markdown_format:
+    executor: pajbot
+    steps:
+      - run:
+          name: Install curl
+          command: apt-get update && apt-get install curl -y
+      - run:
+          name: Install apt repository for Node.js 12
+          command: curl -sL https://deb.nodesource.com/setup_12.x | bash -
+      - run:
+          name: Install Node.js 12
+          command: apt-get install nodejs -y
+      - checkout
+      - run:
+          name: Check markdown style with prettier
+          command: npx prettier@^1.18.2 --check --prose-wrap=always **/*.md
```

Comment:
I'd like the prose-wrap parameter removed, I don't like the line-wrapping things it does.
Also, moving this to a script under `scripts/` would be dank, so you can just run this before committing (or even put it in your pre-commit hook!)